cmd/minishift/cmd/hostfolder/util.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hostfolder
+
+import (
+	"fmt"
+	cmdConfig "github.com/minishift/minishift/cmd/minishift/cmd/config"
+	"github.com/minishift/minishift/pkg/minishift/config"
+	"github.com/minishift/minishift/pkg/minishift/hostfolder"
+	"github.com/minishift/minishift/pkg/util"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/spf13/viper"
+	"regexp"
+	"strings"
+)
+
+const (
+	HostfoldersDefaultPath  = "/mnt/sda1"
+	HostfoldersMountPathKey = "hostfolders-mountpath"
+)
+
+var (
+	optionsPattern  = regexp.MustCompile(`^([a-z]+=.*,)?([a-z]+=.*)$`)
```

Comment:
I think `optionsPattern/keyValuePattern` need to be renamed to something like `optionsRegex` since when we say pattern I was thinking only regex pattern.

Reply:
I am not following you on this one